app/src/ui/app.tsx

```diff
@@ -347,7 +347,10 @@ export class App extends React.Component<IAppProps, IAppState> {
       cancelable: true,
     })
 
-    if (document.activeElement.dispatchEvent(event)) {
+    if (
+      document!.activeElement &&
```

Comment:
We only need to `!` the elements when they might be `null` and we know they are non-null. I think `document` is fine here, so this can become:

```ts
if (
  document.activeElement &&
  document.activeElement.dispatchEvent(event)
) {
  remote.getCurrentWebContents().selectAll()
}
```

Reply:
ah yes, we can. i mistakenly thought `document` needed to be `!`, but i checked again and that is not the case. sorry about that!